examples/reference/widgets/AutocompleteInput.ipynb

```diff
@@ -44,8 +44,10 @@
     "\n",
     "##### Styling\n",
     "\n",
+    "- **`color` (str)**: One of \"default\" (default), \"primary\", \"secondary\", \"error\", \"info\", \"success\", \"warning\", \"light\", \"dark\" or \"danger\".\n",
```

Comment:
No matter what I set the `color` to the border is blue when the autocompleteinput is active

```python
import panel as pn
import panel_material_ui as pmui

pn.extension()

pmui.FlexBox(
    pmui.AutocompleteInput(label='Default', color="default"),
    pmui.AutocompleteInput(label='Primary', color="primary"),
    pmui.AutocompleteInput(label='Secondary', color="secondary"),
).servable()
```